python/ray/rllib/agents/impala/impala.py

```diff
@@ -42,6 +43,8 @@
     "num_parallel_data_loaders": 1,
     # level of queuing for sampling.
     "max_sample_requests_in_flight_per_worker": 2,
+    # max number of workers to broadcast one set of weights to
+    "broadcast_interval": 5,
```

Comment:
In the original benchmark code this was if `random.random()  > 0.8` but making it deterministic is probably better.